lib/media/track/index.js

```diff
@@ -64,6 +64,14 @@ class Track extends EventEmitter {
   get id() {
     return this._id;
   }
+
+  toJSON() {
```

Comment:
@markandrus As discussed, `toJSON` can be generic so that we only have to define it in the base classes.